### ISSUE TITLE
Project-level annotation groups and styles

### ISSUE THREAD

**Issue by cooperlab** [DigitalSlideArchive/HistomicsUI#21]
Currently users have to manually enter group names for each slide. For projects with a large number of cases, this is very time consuming, and leads to inconsistencies for annotation group names between slides.

In many projects we want to generate and apply a set of annotation groups for all slides. This would save time and improve consistency.

**Comment by manthey** [comment]
We could pull (and save) the defaults to a json config file (`.hui_groups.json`?), and, if not set, load them.  We should also flag if they have been manually changed from such a file, and, if not, reload when the image changes.  If so, perhaps provide a warning.  See #138 for config file reading.

**Comment by manthey** [comment]
I think these files should be a thin wrapper around export from the styles dialog with something like:
```
{
  "replaceAll": <boolean>,
  "groups": <export from style dialog>
}
```
When we first load an image in HistomicsUI, we would call `folder/{parent folder}/json_config/.hui_groups.json`, and if that exists we load it.  If `replaceAll` is true, then this is like using the import in checked mode.

**Comment by manthey** [comment]
This was resolved by #228.
